docs/tutorials/launch-jupyter-lab-tutorial.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
+---
+
+# Jupyter Lab Notebook Tutorial
+
+
+In this tutorial we will show how to open Jupyter Lab notebook
+on your computer using executor rented from fair. This way
+you can leverage your GPU to run inference and avoid paying for expensive
+cloud compute.
```

Comment:
If you're saying that executor needs to be rented - then you cannot say that user avoids paying for the cloud. You can just say "In this tutorial we will show how to open Jupyter Lab notebook using Fair Compute platform" and avoid mentioning cloud compute costs.

Also - I recommend avoid using the word `executor`. It is internal Fair Compute concept. For outside readers it is better to say "GPU" or "machine".